src/mlpack/tests/lmnn_test.cpp

```diff
@@ -357,7 +357,10 @@ BOOST_AUTO_TEST_CASE(LMNNSGDSimpleDatasetTest)
   LMNN<> lmnn(dataset, labels, 1);
 
   arma::mat outputMatrix;
-  lmnn.LearnDistance(outputMatrix);
+  std::stringstream stream;
+
+  lmnn.LearnDistance(outputMatrix, ens::ProgressBar(70, stream));
```

Comment:
I think it would be better to leave the exsisting tests untouched, as we like to test the method without using callbacks as well. Also, the majority of the callbacks tests are in the https://github.com/mlpack/mlpack/blob/master/src/mlpack/tests/callback_test.cpp should we include the ``lmnn``and ``nca`` tests there as well.

Reply:
You are right, I have moved the test :)